docs/guide/quickstart.rst

```diff
@@ -0,0 +1,292 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_quickstart:
+
+Quickstart
+===========
+
+This page will contain a simple explanation of how to make a basic bot using the commands extension "ext.commands".
+Before you begin, make sure to install discord.py - see :ref:`guide_intro_installation` for more information.
+
+Obtaining a Bot Token
+----------------------
+
+You will need a bot application before being able to run a bot. A step-by-step tutorial with screenshots follows:
+
+1. Go to the `Discord Developer Portal <https://discord.com/developers/applications/>`_.
+2. Create a new application if you have not made one already:
+    .. image:: /images/discord_create_app_button.png
```

Comment:
Minor nitpick but since the recent token display changes, this step may need re-worded.

They now *only* show the Token after a regeneration, it is not viewable beyond this stage.